scripts/deploy.ts

```diff
@@ -7,7 +7,7 @@ import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 
 const STARKNET_MESSAGING_CONTRACT =
   "0xae0ee0a63a2ce6baeeffe56e7714fb4efe48d419";
-const INCENTIVES_CONTROLLER = "0x83d055d382f25e6793099713505c68a5c7535a35";
+const INCENTIVES_CONTROLLER = "0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5";
```

Comment:
@hsouf why was it different? probably we should create a file addresses.ts (either in `scripts/` or in `test`) and import them in deploy scripts and test files

Reply:
the older one is the implementation address, updated it to the proxy address instead

Reply:
But i'm still getting this weird error when deploying the l1 bridge at `deployL1Bridge()`: 
```
Error: Transaction reverted: function returned an unexpected amount of data
    at Bridge.initialize (contracts/l1/Bridge.sol:68)
```